front/src/modules/timetableItem/components/ImportTimetableItem/helpers/parseXML.ts

```diff
@@ -72,6 +125,197 @@ export const getMostFrequentInterval = (schedules: TrainSchedule[]): Duration =>
+    let minTimeDifference = Infinity;
+
+    // Only consider unmatched schedules
+    const availableViriatoSchedules = sortedViriatoSchedules.filter(
+      (schedule) => !matchedViriatoScheduleKeys.has(getScheduleKey(schedule))
+    );
+
+    availableViriatoSchedules.forEach((viriatoSchedule) => {
+      const viriatoDepartureTime = new Date(viriatoSchedule.start_time);
+      const timeDifferenceMs = Math.abs(
+        osrdOccurrence.startTime.getTime() - viriatoDepartureTime.getTime()
+      );
+      const timeDifferenceMinutes = timeDifferenceMs / (1000 * 60);
+
+      if (
+        timeDifferenceMinutes <= intervalMinutes / 4 &&
+        timeDifferenceMinutes < minTimeDifference
+      ) {
+        bestCandidate = viriatoSchedule;
+        minTimeDifference = timeDifferenceMinutes;
+      }
+    });
+
+    if (bestCandidate) {
+      osrdOccurrence.matchedViriatoSchedule = bestCandidate;
+      matchedViriatoScheduleKeys.add(getScheduleKey(bestCandidate));
+    }
+  });
+
+  const exceptions: PacedTrainException[] = [];
+  osrdDefaultOccurrences.forEach((osrdOccurrence) => {
+    if (!osrdOccurrence.matchedViriatoSchedule) {
+      // Case 1: OSRD default occurrence has no correspondence in Viriato file
+      // Create a disabled exception
+      exceptions.push({
+        key: uuidV4(),
+        occurrence_index: osrdOccurrence.index,
+        disabled: true,
+      });
+    } else {
+      // Case 2: OSRD default occurrence has a correspondence in Viriato file
+      const { isIdenticalToModel, differences } = getScheduleDifferences(
+        {
+          ...modelTrainSchedule,
+          start_time: osrdOccurrence.startTime.toISOString(),
+        },
+        osrdOccurrence.matchedViriatoSchedule
+      );
+
+      if (!isIdenticalToModel) {
+        const exception: PacedTrainException = {
+          key: uuidV4(),
+          occurrence_index: osrdOccurrence.index,
+        };
+        applyScheduleDifferencesToException(
+          exception,
+          osrdOccurrence.matchedViriatoSchedule,
+          differences
+        );
+        exceptions.push(exception);
+      }
+    }
+  });
+
+  // Processing Unattributed Trains as added exceptions
+  const allViriatoScheduleKeys = new Set(sortedViriatoSchedules.map(getScheduleKey));
+  const unmatchedViriatoScheduleKeys = new Set(
+    Array.from(allViriatoScheduleKeys).filter((key) => !matchedViriatoScheduleKeys.has(key))
+  );
+
+  sortedViriatoSchedules.forEach((viriatoSchedule) => {
+    if (unmatchedViriatoScheduleKeys.has(getScheduleKey(viriatoSchedule))) {
+      const { isIdenticalToModel, differences } = getScheduleDifferences(
+        modelTrainSchedule,
+        viriatoSchedule
+      );
+
+      const exception: PacedTrainException = {
+        key: uuidV4(),
+      };
+
+      if (!isIdenticalToModel) {
+        applyScheduleDifferencesToException(exception, viriatoSchedule, differences);
+      }
+
+      exceptions.push(exception);
+    }
```

Comment:
this part breaks the app when importing some files, as we're always pushing exceptions even when the trainSchedule is identical to the model. In `useOccurences`, the exception expects a `start_time!.value` and here we're not providing it. 
I tried by preventing the exception to be pushed and it seems to work : 
```suggestion

if (!isIdenticalToModel) {
  const exception: PacedTrainException = {
    key: uuidV4(),
  };

  applyScheduleDifferencesToException(exception, viriatoSchedule, differences);
  exceptions.push(exception);
}

```

another solution could be to provide the exception with the start_time of the trainSchedule ?

Reply:
Done :)